src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -300,7 +300,7 @@ private async Task<Recommendation> GetSelectedRecommendationFromPreviousDeployme
                             return false;
 
                         var value = selectedRecommendation.GetOptionSettingValue(x);
-                        if (value == null || value.ToString() == string.Empty || object.Equals(value, x.DefaultValue))
+                        if (value == null || value.ToString() == string.Empty)

```

Comment:
I made this change because when viewing previous settings for a deployment it hides a lot of the current settings because the value matches the default setting.

Reply:
Not sure what the comment would be for absence of code. Especially in the long term that we in the past used to have a check and now we don't.